cram_3d_world/cram_urdf_projection/cram_urdf_projection/src/process-modules.lisp

```diff
@@ -0,0 +1,104 @@
+;;;
+;;; Copyright (c) 2017, Gayane Kazhoyan <kazhoyan@cs.uni-bremen.de>
+;;;               2019, Vanessa Hassouna <hassouna@uni-bremen.de>
```

Comment:
Hm, I don't see any substantial contributions from you in this file, so I think it doesn't make sense to put your name here. I do see contributions from you in low-level.lisp though, maybe that's what you wanted to do in the first place.